src/turbopuffer.test.ts

```diff
@@ -1,12 +1,15 @@
-import { Turbopuffer, TurbopufferError } from "./turbopuffer";
+import { Turbopuffer } from "./turbopuffer";
+import { isRuntimeFullyNodeCompatible, TurbopufferError } from "./helpers";
 
 const tpuf = new Turbopuffer({
   apiKey: process.env.TURBOPUFFER_API_KEY!,
 });
 
+const testNamespacePrefix = "typescript_sdk_";
```

Comment:
btw extracted the string literal into this variable primarily because of Bun (`bun test`): https://github.com/oven-sh/bun/issues/6213